remediation/remediation.md

```diff
@@ -52,8 +55,9 @@ single cluster, without creating conflicts.
 ## Implementations
 
 Implementations conforming to the ExternalRemediation API include:
+
 * [self-node-remediation](/remediation/self-node-remediation/self-node-remediation/) - a mechanism designed for shared-nothing environments without programmatic access to BMC-like hardware
 * [(wip) metal3](https://github.com/metal3-io/cluster-api-provider-metal3/pull/157) - a mechanism designed for bare metal clusters with a functioning Metal3 API
 * [machine-deletion-remediation](https://github.com/medik8s/machine-deletion-remediation) - a mechanism designed for any cluster with a functioning Machine API
-* [(planned) direct]() - a mechanism designed around an existing set of [upstream fencing agents](https://github.com/ClusterLabs/fence-agents) for environments with a traditional API end-point (eg. IPMI) for power cycling cluster nodes
+* [fence-agents-remediation](https://github.com/medik8s/fence-agents-remediation) - a mechanism designed around an existing set of [upstream fencing agents](https://github.com/ClusterLabs/fence-agents) for environments with a traditional API end-point (eg. IPMI) for power-cycling the cluster nodes
```

Comment:
Do we want to prioritize FAR and rank it higher in this list and in [navigation bar](https://github.com/medik8s/docs/pull/63/files#diff-292f5d801f6deb757dcd72b8b27a9cb338cf46664452fdb4b91016ba4aece3d8R4)?

Reply:
We agreed internally, that there is no meaning for this order here. It is unordered list